javalanglib/src/main/scala/java/lang/Thread.scala

```diff
@@ -20,10 +20,24 @@ class Thread private (dummy: Unit) extends Runnable {
     scala.scalajs.runtime.StackTrace.getCurrentStackTrace()
 
   def getId(): scala.Long = 1
+
+  private var interruptedState = false
+
+  def interrupt(): Unit =
+    interruptedState = true
+
+  def isInterrupted(): scala.Boolean =
+    interruptedState
```

Comment:
To match the JavaDoc's order, these two methods should be put between `run()` and `setName()`.